daemon/policy.go

```diff
@@ -104,14 +104,11 @@ func (d *Daemon) UpdatePolicyEnforcement(e *endpoint.Endpoint) bool {
 	if d.conf.EnablePolicy == endpoint.AlwaysEnforce {
 		return true
 	} else if d.conf.EnablePolicy == endpoint.DefaultEnforcement && !d.conf.IsK8sEnabled() {
-		log.Infof("updatePolicyEnforcement: default enforcement, no k8s")
```

Comment:
These were debugging messages I had left behind, so I removed them to clean up the logs a bit.